neo/app/controllers/Application.scala

```diff
@@ -59,9 +59,11 @@ class Application @Inject()(val wsClient: WSClient,
     ActorFlow.actorRef(out => UserActor.props(out, cachesActor, USGeoGnosis))
   }
 
-  def tweet(id:String) = Action {
+  def tweet(id:String) = Action.async {
```

Comment:
I'm very happy to explain Scala code in review comment, but I really don't like comments in the code which usually shows that the code is not clear enough 😢 

This function actually is a Play framework function which sends a request to that composed `url`. `wsClient.url().get` get the response back by return a `Future[WSResponse]`. Then we just wrap up the body of the response with a "OK 200" return code and send back to the client.